moon/gm_mixpanel/base.moon

```diff
@@ -34,6 +34,10 @@ class MixpanelBase
         @MAX_QUEUE_SIZE = 50
         @eventQueue = {}
 
+    _getToken: => getToken!
+
+    _getTimestamp: => getTimestamp!
+
```

Comment:
Not really following why you did this?

Reply:
The other classes inherit this, but these values are defined locally in this file